inference-engine/tests_deprecated/behavior/vpu/myriad_tests/vpu_protocol_tests.cpp

```diff
@@ -26,6 +27,33 @@ TEST_P(MyriadProtocolTests, CanInferenceWithProtocol) {
     ASSERT_EQ(statusCode, StatusCode::OK) << resp.msg;
 }
 
+
+
+TEST_P(MyriadProtocolTests, NoWarningIfPatchFirmwareForUSBDevice) {
```

Comment:
The name was a little confusing. I suggest something like "NoErrorsMessagesWhenLoadNetworkSuccessful"